provider/core/src/hello_world.rs

```diff
@@ -130,12 +135,17 @@ impl DataPayload<HelloWorldV1Marker> {
     }
 }
 
-#[cfg(not(feature = "datagen"))]
-impl_dynamic_data_provider!(HelloWorldProvider, [HelloWorldV1Marker,], AnyMarker);
+/// An AnyProvider supporting the `HelloWorldV1` key.
+pub struct HelloWorldAnyProvider;
```

Comment:
Sure, but it supported AnyProvider before, using the macro to generate the impl. Why change? (Besides that, it's breaking)

Reply:
Reverted